packages/bloc_test/pubspec.yaml

```diff
@@ -9,7 +9,8 @@ environment:
   sdk: ">=2.6.0 <3.0.0"
 
 dependencies:
-  bloc: ^3.0.0
+  bloc:
```

Comment:
why is this change necessary? I'd prefer to keep it as is since we cannot publish with path imports

Reply:
Oh right, my thinking was to make sure to test cross-packages dependencies breaking changes in pull requests.
Thanks for guiding my through all the rookie mistakes :-)